### PR TITLE
Make mobile version of EventSchedule

### PR DESCRIPTION
**Pull request by rosslh**
## Proposed Change

Fixes: #107 
Fixes: #120 

Make mobile version of EventSchedule!

### How did you do this?

- I used react-responsive which conditionally renders one of two components
  - EventScheduleCompactTable
  - EventScheduleFullSizeTable

### Why are you choosing this approach?

- Since the structure and content of the tables changes, it's not possible to change the tables purely with CSS

### Any open questions you want to ask code reviewers?

Nope

### List of changes:

- Add EventScheduleCompactTable component
- Add EventScheduleFullSizeTable component
- Use react-responsive to render one of those as a child of EventSchedule
- Set default font-weight to 500 to match mockups
- Add and update tests

## Pull Request Checklist:

  - [x] My submission passes all tests.
  - [x] I have lint my code locally prior to submission.
  - [x] I have written new tests for my core changes (where applicable).
  - [x] I have referenced all useful information (issues, tasks, etc) in the pull request.

### CODE REVIEW

**Comment by RobertWSaunders** [comment]
Deploy preview for *qhacks-website* ready!

Built with commit 626e1a9ecaa3c0710cf0844b552c9c3d132b81bf

https://deploy-preview-129--qhacks-website.netlify.com

**Comment by rosslh** [comment]
I know the heading font-size isn't changing yet, but I figured that might be something we'll want to do globally at a certain breakpoint.

**Comment by joeytepp** [comment]
@rosslh I agree definitely makes sense for this component

File: package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "react": "^16.5.0",
     "react-dom": "^16.5.0",
     "react-helmet": "^5.2.0",
+    "react-responsive": "^5.0.0",
```
Comment (joeytepp) [inline package.json]: Nice 👌

File: src/components/EventSchedule.js [hunk 2]
```diff
@@ -192,45 +182,23 @@ class EventSchedule extends Component {
...
-            >
-              <th>Time</th>
-              <th>Duration</th>
-              <th>Event</th>
-              <th>Location</th>
-            </tr>
-            {rows}
-          </tbody>
-        </table>
+        <MediaQuery query="(min-device-width: 700px)">
```
Comment (joeytepp) [inline src/components/EventSchedule.js]: This looks cool should probably make an issue once this merges to update our other media queries where applicable.

File: src/components/EventScheduleCompressedTable.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { css } from "glamor";
+import moment from "moment-timezone";
+
+moment.tz.setDefault("America/Kingston");
+
+const EventScheduleFullSizeTable = (props) => (
```
[the remark this answers is not in the document]
Comment (RobertWSaunders) [reply src/components/EventScheduleCompressedTable.js]: Reason that I'm calling you out here is cause you gave the approval for this but if that was the case you should have caught this.

[the remark this answers is not in the document]
Comment (RobertWSaunders) [reply src/components/EventScheduleCompressedTable.js]: I don't really mind but in the summer I was told that it looks pretty poorly on you if you approve something and something silly like this gets caught after the fact, shows you didn't fully read it or have attention to detail.

File: src/components/EventScheduleCompressedTable.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { css } from "glamor";
+import moment from "moment-timezone";
+
+moment.tz.setDefault("America/Kingston");
```
[the remark this answers is not in the document]
Comment (rosslh) [reply src/components/EventScheduleCompressedTable.js]: Yeah, looked into it and we can just set it once since it applies globally

**Comment by rosslh** [comment]
@RobertWSaunders I made changes based on your feedback

File: src/components/EventScheduleFullSizeTable.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { css } from "glamor";
+import moment from "moment-timezone";
+
+const EventScheduleFullSizeTable = (props) => (
+  <table
+    {...css({
```
Comment (RobertWSaunders) [inline src/components/EventScheduleFullSizeTable.js]: It appears that the table still needs the border around it?

**Pull request merged**